server/views/partials/visualise.hbs

```diff
@@ -27,89 +27,197 @@
 
 	</div>
 
-</main>
 
-<script>
+	<script>
+
+		var table = document.querySelector('table'),
+			ths = table.querySelectorAll('thead th'),
+			rows = table.querySelectorAll('tbody tr');
 
-	var table = document.querySelector('table'),
-		ths = table.querySelectorAll('thead th'),
-		rows = table.querySelectorAll('tbody tr');
+		function ascendingSort(a, b){
+
+			if(a < b){
+				return -1;
+			} else if (b < a){
+				return 1;
+			} else {
+				return 0;
+			}
+
+		}
 
-	function ascendingSort(a, b){
+		function descendingSort(a, b){
 
-		if(a < b){
-			return -1;
-		} else if (b < a){
-			return 1;
-		} else {
-			return 0;
+			if(a < b){
+				return 1;
+			} else if (b < a){
+				return -1;
+			} else {
+				return 0;
+			}
+			
 		}
 
-	}
+		function sortRowsByColumn(idx, ASC, isNumberValue){
+
+			var r = Array.from(rows),
+				tbody = table.querySelector('tbody');
+
+			r.sort(function(a, b){
 
-	function descendingSort(a, b){
+				var aCol = a.childNodes[idx].textContent;
+				var bCol = b.childNodes[idx].textContent;
+
+				if(isNumberValue){
+					aCol = parseInt(aCol);
+					bCol = parseInt(bCol);
+				}
+
+				if(ASC){
+					return ascendingSort(aCol, bCol);
+				} else {
+					return descendingSort(aCol, bCol);
+				}
+			
+			});
+
+			tbody.innerHTML = "";
+
+			r.forEach(function(row){
+				tbody.appendChild(row);
+			});
 
-		if(a < b){
-			return 1;
-		} else if (b < a){
-			return -1;
-		} else {
-			return 0;
 		}
+
+		Array.from(ths).forEach(function(th, idx){
+
+			(function(columnIndex){
```

Comment:
Specifically, turning the click handling function on line 96 into an arrow function